src/EA.Weee.Web/Areas/AATFReturn/Requests/ObligatedReceivedWeeeRequestCreator.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Requests
+{
+    using System;
+    using System.Collections.Generic;
+    using ViewModels;
+    using Web.Requests.Base;
+    using Weee.Requests.AatfReturn.ObligatedReceived;
+
+    public class ObligatedReceivedWeeeRequestCreator : RequestCreator<ObligatedReceivedViewModel, AddObligatedReceived>, IObligatedReceivedWeeeRequestCreator
+    {
+        public override AddObligatedReceived ViewModelToRequest(ObligatedReceivedViewModel viewModel)
```

Comment:
Can we do a Guard here to check for NULL view model parameter and equivalent unit test, think there may be other request creators we need to do this for